tests/integration/llm/test_llm_cli_usage.py

```
@@ -0,0 +1,214 @@
+def test_llm_prompt_streams_output() -> None:
+    """验证默认流式输出可逐步读取。
+
+    Note: 在非TTY管道环境（CI常见），llm CLI可能一次性刷新输出而非流式。
+    此测试兼容两种场景：优先验证流式行为，但允许批量输出。
+    """
+    prompt = (
+        "请分三步输出内容，每步单独换行：第一行写'开始'，"
+        "第二行写数字 1,2,3（每个独占一行），第三行写'结束'。"
+    )
+
+    process = subprocess.Popen(
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)